test/src/harness_wind.hpp

```diff
@@ -26,7 +26,7 @@
 
 #include <stdio.h>
 
-#define NUM_WIND 9
+static constexpr int NUM_WIND = 9;
```

Comment:
This one is still signed. Of course, the existing code is crap and that makes your life harder than necessary; once you migrate from macros to constexpr, some things are suddenly explicit and you can't get away with the wrong types.